esp/services/ws_configmgr/ws_configmgrService.cpp

```diff
@@ -352,7 +355,7 @@ bool Cws_configMgrEx::onGetNode(IEspContext &context, IEspGetNodeRequest &req, I
     }
 
     getNodeResponse(pNode, resp);
-    pNode->validate(status, false);  // validate this node only
+    pNode->getSchemaItem()->validate(status, false);  // validate this node only
```

Comment:
Can schemaItem be a nullptr?

Reply:
No. I ensure that all nodes have a schema item. This is even the case if no schema is defined. A default schema item is constructed for this case.